python/torch_mlir_e2e_test/torchscript/configs/lazy_tensor_core.py

```diff
@@ -23,12 +23,33 @@ def run(self, artifact: torch.nn.Module, trace: Trace) -> Trace:
 
         for item in trace:
             # We need to move all the inputs to the lazy device before running in LTC.
-            lazy_inputs = [arg.to('lazy') for arg in item.inputs]
+            lazy_inputs = to_device(item.inputs, device='lazy')
             output = getattr(artifact, item.symbol)(*lazy_inputs)
+            cpu_outputs = to_device(output, device='cpu')
 
             result.append(
                 TraceItem(symbol=item.symbol,
                           inputs=item.inputs,
-                          output=output.to('cpu')))
+                          output=cpu_outputs))
 
         return result
+
+
+def to_device(value, device):
+    """
```

Comment:
recommend using tree_map from `from torch.utils._pytree import tree_map` -- see example use in python/torch_mlir_e2e_test/torchscript/configs/eager_mode.py